applications/static/main.tf

```diff
@@ -10,6 +10,64 @@ variable "stack" {
   description = "The 'stack' for this bucket: web, sms, backend, data."
 }
 
+resource "fastly_service_v1" "cdn" {
+  name          = "Terraform: ${var.domain}"
+  force_destroy = true
+
+  domain {
+    name = var.domain
+  }
+
+  backend {
+    name    = "s3-${var.domain}"
+    address = "${aws_s3_bucket.bucket.id}.s3-website-${aws_s3_bucket.bucket.region}.amazonaws.com"
+    port    = 80
+  }
+
+  request_setting {
+    name      = "Force SSL"
+    force_ssl = true
+  }
+
+  gzip {
+    name = "gzip"
+
+    extensions = ["css", "js", "html", "eot", "ico", "otf", "ttf", "json"]
+
+    content_types = [
+      "text/html",
+      "application/x-javascript",
+      "text/css",
+      "application/javascript",
+      "text/javascript",
+      "application/json",
+      "application/vnd.ms-fontobject",
+      "application/x-font-opentype",
+      "application/x-font-truetype",
+      "application/x-font-ttf",
+      "application/xml",
+      "font/eot",
+      "font/opentype",
+      "font/otf",
+      "image/svg+xml",
+      "image/vnd.microsoft.icon",
+      "text/plain",
+      "text/xml",
+    ]
+  }
+
+  # The S3 backend only returns a 'Vary' header if a request has a CORS header on it, which
+  # means we may accidentally cache a CORS-less response for everyone. This rule adds the
+  # expected 'Vary' if it isn't already set on the response.
+  header {
+    name        = "S3 Vary"
+    type        = "cache"
+    action      = "set"
+    destination = "http.Vary"
+    source      = "\"Origin, Access-Control-Request-Headers, Access-Control-Request-Method\""
+  }
+}
+
```

Comment:
This is extracted from the production `fastly-frontend` module (below). Since we now only handle one domain in each of these two properties, we no longer need request conditions for backends/headers.

Reply:
I don't see anything about `assets.dosomething.org` specifically referenced here like I do in `fastly-frontend` down below, can you explain this a little more?

Reply:
Yeah, totes! We read the domain that the Fastly property & S3 bucket will be configured for using the `var.domain` variable in this module (declared at the top of this file, and used in the Fastly `domain { … }` block & bucket config).

When using this module to create resources in `voting-app/main.tf` and `dosomething/main.tf`, we can then pass a different domain (either `athletesgonegood.com` or `assets.dosomething.org`) to customize each "instance".

Reply:
Ah I think I see now, and I do see those being passing through elsewhere!